EpikCord/close_event_codes.py

```diff
@@ -4,6 +4,8 @@
 class GatewayCECode(IntEnum):
     """Gateway close event code."""
 
+    WEBSOCKET_ERROR = 1006
```

Comment:
The proper name for this is `UNUSUAL CLOSURE` or somethin,

Reply:
*CLOSE_ABNORMAL: No close code frame has been receieved*